features/approve-member.feature

```diff
@@ -29,9 +29,9 @@ Feature: As an admin
 
     And the following applications exist:
       | first_name | user_email            | company_number | status  | category_name |
-      | Emma       | emma@happymutts.se    | 5562252998     | pending | Rehab         |
-      | Hans       | hans@happymutts.se    | 5562252998     | pending | grooming      |
-      | Anna       | anna@nosnarkybarky.se | 5560360793     | pending | Rehab         |
+      | Emma       | emma@happymutts.se    | 5562252998     | Pending | rehab         |
```

Comment:
This is because the business categories (defined in a table above this one) have lower-case names.  The names need to match, since - in the membership application fixture step - we look up the category by name and associated it with the membership application.